WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -43,6 +43,6 @@ class JetpackWindowManager: WindowManager {
 
     // TODO: Add logic in here to trigger migration UI if needed
     private var shouldShowMigrationUI: Bool {
-        false
+        return FeatureFlag.contentMigration.enabled
```

Comment:
1. I think we want to ensure that the app is in a logged-out state here. I _think_ you can do that via `AccountHelper.isLoggedIn`, but I'm not sure if that method already covers the case of self-hosted sites.
2. Additionally, we should only start the migration flow after the user taps the "Try the new Jetpack app" button (i.e., the `/app` deep linking). This means we'll need to set some state when JP receives the `/app` intent so this `shouldShowMigrationUI` can check against that.

What do you think?

Reply:
Oh, I just read the entire `JetpackWindowManager` and it looks like the login requirement has been handled. My bad! 😅 But what do you think about the deeplink part?

Reply:
@dvdchr When I was working on this PR, I wasn't aware of the "`/app` deep linking" changes.  So the purpose of this PR was only to use the "Content Migration" flag. That's why I kept this todo:

```
// TODO: Add logic in here to trigger migration UI if needed
```

I'm planning to open another PR implementing the exact Migration conditions.

It would be great to move forward with this PR ( if it's okey with you ), because it's blocking this one:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/19662

Reply:
I also noticed that this PR was showing way more File Changes than expected. Only 1 file is changed. I fixed that. 🙏